pkg/util/fedinformer/transform.go

```diff
@@ -12,6 +13,10 @@ import (
 // StripUnusedFields is the transform function for shared informers,
 // it removes unused fields from objects before they are stored in the cache to save memory.
 func StripUnusedFields(obj interface{}) (interface{}, error) {
+	if tombstone, ok := obj.(cache.DeletedFinalStateUnknown); ok {
+		obj = tombstone.Obj
+	}
+
```

Comment:
What does this code mean? Should we proceed or ignore it?

Reply:
If the object is a DeletedFinalStateUnknown, we need to extract the actual object and proceed it